_config.yml

```diff
@@ -1,8 +0,0 @@
-theme: jekyll-theme-tactile
-plugins:
-  - jekyll-relative-links
-relative_links:
-  enabled: true
-  collections: true
-include:
-  -README.md
```

Comment:
Deze file wordt gebruikt voor github pages. die hoeft niet weg. (als die lastig terug te zetten is, stel ik het wel opnieuw in na de merge